server/api/rounds.py

```diff
@@ -720,13 +720,42 @@ def validate_round(round: dict, election: Election):
     if current_round and not current_round.ended_at:
         raise Conflict("The current round is not complete")
 
-    if round["roundNum"] == 1:
-        if "sampleSizes" not in round:
-            raise BadRequest("Sample sizes are required for round 1")
 
-        targeted_contest_ids = {c.id for c in election.contests if c.is_targeted}
-        if set(round["sampleSizes"].keys()) != targeted_contest_ids:
-            raise BadRequest("Sample sizes provided do not match targeted contest ids")
+def validate_custom_sample_size(round: dict, election: Election):
```

Comment:
`validate_sample_sizes` would be more accurate, since it's not only custom, and there are potentially multiple sample sizes